validators.go

```diff
@@ -27,8 +27,8 @@ import (
 )
 
 var (
-	upperAlphanumericRegex = regexp.MustCompile(`[^ A-Z0-9!"#$%&'()*+,-.\\/:;<>=?@\[\]^_{}|~]+`)
-	alphanumericRegex      = regexp.MustCompile(`[^ \w!"#$%&'()*+,-.\\/:;<>=?@\[\]^_{}|~]+`)
+	upperAlphanumericRegex = regexp.MustCompile(`[^ A-Z0-9!"#$%&'()*+,-.\\/:;<>=?@\[\]^_{}|~` + "`" + `]+`)
```

Comment:
Using `fmt.Sprintf` should work here too:
```go
	upperAlphanumericRegex = regexp.MustCompile(fmt.Sprintf(
		`[^ A-Z0-9!"#$%%&'()*+,-.\\/:;<>=?@\[\]^_{}|~%s]+`, "`",
	))

```

Reply:
Seems a little more readable, I'll adopt this.